src/carousel/carousel.ts

```diff
@@ -83,7 +84,10 @@ export class NgbSlide {
     <ol class="carousel-indicators" [class.visually-hidden]="!showNavigationIndicators" role="tablist">
       <li *ngFor="let slide of slides" [class.active]="slide.id === activeId"
           role="tab" [attr.aria-labelledby]="'slide-' + slide.id" [attr.aria-controls]="'slide-' + slide.id"
+          [attr.data-bs-target]="id"
+          [attr.data-bs-to]="'slide-' + slide.id"
           [attr.aria-selected]="slide.id === activeId"
+          [attr.aria-curent]="slide.id === activeId"
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current :

> When something is selected rather than current, such as a [tab](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) in a [tablist](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role), use [aria-selected](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) to indicate the currently-displayed tabpanel.

Unlike bootstrap, we are using `role="tablist"` and `role="tab"` for carousel indicators, because they are somehow similar to tabs in a tablist. So is it correct to use both `aria-selected` and `aria-current`?